NightscoutUploadKit/NightscoutUploader.swift

```diff
@@ -277,8 +298,77 @@ public class NightscoutUploader {
             completion(.failure(UploadError.invalidParameters))
         }
     }
+    
+    
+    // MARK: - Loop Notifications
+
+    public func startOverride(overrideName: String, reasonDisplay: String, durationInMinutes: Int) async throws {
+
```

Comment:
I did not make a model for these different payloads as I was not sure if it is worth it for this 1.0 remote functionality. Happy to if desired.

Reply:
I think this is fine. As long as the exposed methods are not expecting to be passed json dicts, or returning json dicts. They should implement whatever niceties they can to provide a "Swifty" API to users of the framework, following Swift idioms and conventions as much as makes sense.

Some of these are taking `durationInMinutes` or `absorptionInHours`.  While I like expressing the expected units of parameters, it might be better to just do a TimeInterval?